### ISSUE TITLE
namespaced/keywords instead of namespaced:keywords

### ISSUE THREAD

**Issue by phiweger** [metasoarous/tripl#9]
my argument is readibility: when you specify a dict, then there are many ":" which (imho) decreases readibility, compare

```
        'mock:type': 'mock.type:seq',
        'mock.seq:id': 'a1',
        'mock.seq:string': 'ACTGA',
        'mock:description': 'some foo from bar',
```

with

```
        'mock/type': 'mock.type:seq',
        'mock.seq/id': 'a1',
        'mock.seq/string': 'ACTGA',
        'mock/description': 'some foo from bar',
```

**Comment by metasoarous** [comment]
OK; Noted. I agree that `/` is more readable, and like that it matches with Datomic/Clojure. However, `:` matches traditional RDF usage more directly, so this should be taken into consideration. We'll definitely have to settle on something.

**Comment by phiweger** [comment]
are there any downstream consequences for either option? (like interop issues)

**Comment by metasoarous** [comment]
Yes; Certainly interop is a consideration. Will users be interfacing more with traditional RDF stores, or databases like Datomic/DataScript/Datsync(Replikativ)? But that's easy to work around. The more subtle but important consequence is in terms of how we explain these ideas to folks. And there we're sort of split, because we're espousing all these great things from RDF, but taking Datomic's design decisions for usability. So it's a little hard to say.

**Comment by metasoarous** [comment]
The other option is to just use dots everywhere, since this is a python library, and that's how python namespaced. So instead of `'mock.seq:description'` or `'mock.seq/description'`, it would just be `'mock.seq.description'`.

**Comment by phiweger** [comment]
I like the entity:value separation in the namespace, which the "only-dots" syntax of `mock.seq.description` blurs, I mean `what.keeps.people.from.doing:that` ;)

what do you think of `mock-seq.description`?
